### PR TITLE
merge the RBACs for monitoring viewers

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform 

After merge:

```console
oc delete ClusterRoleBinding ci-monitoring-view --context app.ci
```

@wking is added into the rover group.

I cannot see YuviGold in github openshift org: https://github.com/orgs/openshift/people
Either in rover.

### CODE REVIEW

File: core-services/sync-rover-groups/_config.yaml [hunk 1]
```diff
@@ -1,11 +1,19 @@
 cluster_groups:
+  build-farm:
```
Comment (smg247) [inline core-services/sync-rover-groups/_config.yaml:2]: is this a spot that `cluster-init` now needs to be aware of? if so can we make a card to handle that?

Comment (hongkailiu) [reply core-services/sync-rover-groups/_config.yaml:2]: I will make a card for this.

**Comment by hongkailiu** [comment]
```console
$ oc --context app.ci delete group cluster-monitoring-viewer --as system:admin
group.user.openshift.io "cluster-monitoring-viewer" deleted

$ oc delete ClusterRoleBinding ci-monitoring-view --context app.ci --as system:admin
clusterrolebinding.rbac.authorization.k8s.io "ci-monitoring-view" deleted
```

**Pull request merged**